fastlane/Fastfile

```diff
@@ -42,12 +42,15 @@ private_lane :merged_prs_since_last_release do |options|
     PullRequest.new(pr["number"], pr["title"], pr["user"]["login"], pr["merge_commit_sha"], pr["merged_at"], pr["body"])
   }
 
-  # Only include PRs that actually made it into the git history (see https://github.com/guardian/android-news-app/pull/5339)
+  current_branch = sh("git branch --show-current")
```

Comment:
This change broke the beta process for us :(
Apparently the git version in the TeamCity VMs is <2.22, which means that `--show-current` is not a valid flag and the script breaks at this point.

I'll create a new PR replacing this with slightly more durable `git rev-parse --abbrev-ref HEAD` and then test our builds.
@SLambrakis maybe you can also help then to ensure that iOS builds continue to work as well.

Reply:
Agh, I'm sorry! I didn't think about TeamCity having a different setup to GH Actions. Yes, please go ahead and make the new PR, and I'll check/deal with iOS once Android is fixed.